howto/setup-pub-sub-message-broker/setup-nats-streaming.md

```diff
@@ -0,0 +1,91 @@
+# Setup NATS
+
+## Locally
+
+You can run a NATS server locally using Docker:
+
+```bash
+docker run -d -name nats-streaming -p 4222:4222 -p 8222:8222 nats-streaming
+```
+
+You can then interact with the server using the client port: `localhost:4222`.
+
+## Kubernetes
+
+Install NATS on Kubernetes by using the [kubectl](https://docs.nats.io/nats-on-kubernetes/minimal-setup):
+
+```bash
+# Single server NATS
+
+kubectl apply -f https://raw.githubusercontent.com/nats-io/k8s/master/nats-server/single-server-nats.yml
+
+kubectl apply -f https://raw.githubusercontent.com/nats-io/k8s/master/nats-streaming-server/single-server-stan.yml
+```
+
+This will install a single NATS-Streaming and Nats into the `default` namespace.
+To interact with NATS, find the service with: `kubectl get svc stan`.
+
+For example, if installing using the example above, the NATS server client address would be:
+
+`<YOUR-HOST>:4222`
```

Comment:
The port now is `7777`

Reply:
![image](https://user-images.githubusercontent.com/32089134/92557459-d45f6e80-f29e-11ea-9550-b520552abf08.png)

The kubectl YAML file shows that 7777 is a metric port. Should I change to 7777?

Reply:
Oh right, keep as 4222

Reply:
Ok